controllers/applicationvolumereplication_controller.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2021 The RamenDR authors.
```

Comment:
(nit) Copyright spaces missing, just attempting to keep things consistent across files.

Reply:
done